os/board/rtl8721csm/src/component/common/api/example_eap.c

```diff
@@ -0,0 +1,238 @@
+"BQAwgZMxCzAJBgNVBAYTAkZSMQ8wDQYDVQQIDAZSYWRpdXMxEjAQBgNVBAcMCVNv\r\n" \
+"bWV3aGVyZTEVMBMGA1UECgwMRXhhbXBsZSBJbmMuMSAwHgYJKoZIhvcNAQkBFhFh\r\n" \
+"ZG1pbkBleGFtcGxlLm9yZzEmMCQGA1UEAwwdRXhhbXBsZSBDZXJ0aWZpY2F0ZSBB\r\n" \
+"dXRob3JpdHkwHhcNMjIwMzE1MTE0MDUwWhcNMjIwNTE0MTE0MDUwWjCBkzELMAkG\r\n" \
+"A1UEBhMCRlIxDzANBgNVBAgMBlJhZGl1czESMBAGA1UEBwwJU29tZXdoZXJlMRUw\r\n" \
+"EwYDVQQKDAxFeGFtcGxlIEluYy4xIDAeBgkqhkiG9w0BCQEWEWFkbWluQGV4YW1w\r\n" \
+"bGUub3JnMSYwJAYDVQQDDB1FeGFtcGxlIENlcnRpZmljYXRlIEF1dGhvcml0eTCC\r\n" \
+"ASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAMxUUxSmkmqlfW71nV4DRrW5\r\n" \
+"zS5yU2jPD+YzGVByKec+WIYsGe8J1+AK5wb7smzvddoH+j5GLf7/9BwuPcwvH6at\r\n" \
+"mIFg7Z+1WyEqAzCXOUSbHnaYJPI6MAmGOLpmRbLe4z350qwQy+XTMid+Z751TX3u\r\n" \
+"mmG3X6t0Toczkkb3kyUmvi9mEVFXKPAI9q87MHF22BqhCNsoJp/EPVJmYbyun1DB\r\n" \
+"AsbOljjQnyvuVvtN0gszGAFPOVuzfQRN94PLDdaiUtq/Z73bvLhzetDwUVSyCBm\r\n" \
+"g5HVAzApKkudLPCdLxUqRu6MrWGWAkVLN3z92wKdqU6gFoui3VXpgaDENlg5Iwp\r\n" \
+"cCAwEAAaOCAUIwggE+MB0GA1UdDgQWBBRjAk790rh+Ibu2jaQaITxfSZdZsTCB0\r\n" \
+"wYDVR0jBIHLMIHIgBRjAk790rh+Ibu2jaQaITxfSZdZsaGBmaSBljCBkzELMAkG\r\n" \
+"A1UEBhMCRlIxDzANBgNVBAgMBlJhZGl1czESMBAGA1UEBwwJU29tZXdoZXJlMRU\r\n" \
+"wEwYDVQQKDAxFeGFtcGxlIEluYy4xIDAeBgkqhkiG9w0BCQEWEWFkbWluQGV4YW\r\n" \
+"1wbGUub3JnMSYwJAYDVQQDDB1FeGFtcGxlIENlcnRpZmljYXRlIEF1dGhvcml0e\r\n" \
+"YIUDYXAVMiBaDloGdG+PE5SDjlr0z4wDwYDVR0TAQH/BAUwAwEB/zA2BgNVHR8E\r\n" \
+"LzAtMCugKaAnhiVodHRwOi8vd3d3LmV4YW1wbGUub3JnL2V4YW1wbGVfY2EuY3J\r\n" \
+"sMA0GCSqGSIb3DQEBCwUAA4IBAQDCIGCQ088yakGIztULrlnghBBGb3TvGYyaGo\r\n" \
+"mk1vrOK3KigZOMH46Fo3lF3SIYdp32wJHHxtfq35T91JfW8EHRZJfQLezsko8cs\r\n" \
+"UnAcUJ2M5mEqq981dfI7ELytGUD2jZ6haZTo3GuazWzG3E9tTq9BZuQW3HcVBZe\r\n" \
+"YOuxxFM04cUhccPMZ09zR8KADmffZ4+Xvjr2Xd588sZ5nfh5EQhsBSposyA1FTi\r\n" \
+"7+rceDTbI8uIV+gX4/uvhyfpIHZOzZvfKSy2AhdFR3vpwY1MhOy5I/jRwJb/Mp/\r\n" \
+"NRSkc9IzVPN94J345/sxrzWoV5wEiH/Ypcc4pmjzvr8Hun1qWso2zr\r\n" \
+"-----END CERTIFICATE-----\r\n";
+
+	eap_client_cert_len = strlen(eap_client_cert)+1;
+	eap_client_key_len = strlen(eap_client_key)+1;
+	eap_ca_cert_len = strlen(eap_ca_cert)+1;
+}
+
+extern void linkup_handler(rtk_reason_t *reason);
+extern void linkdown_handler(rtk_reason_t *reason);
+
+static void example_eap_thread(void *method){
+#if defined(CONFIG_PLATFORM_TIZENRT_OS)
+	int ret = WiFiRegisterLinkCallback(&linkup_handler, &linkdown_handler);
+	if (ret != RTK_STATUS_SUCCESS) {
+		printf("\n\r%s: [RTK] Link callback handles: register failed !\n", __func__);
+		return -1;
+	} else {
+		printf("\n\r%s: [RTK] Link callback handles: registered\n", __func__);
+	}
+#endif
+
+	example_eap_config();
+	
+	if(strcmp(method, "tls") == 0){
+		// tls must present client_cert, client_key
+		eap_start("tls");
+	}
+	else if(strcmp(method, "peap") == 0){
+		eap_start("peap");
+	}
+	else if(strcmp(method, "ttls") == 0){
+		eap_start("ttls");
+	}
+	else if(strcmp(method, "fast") == 0){
+		eap_start("fast");
+	}
+	else
+		printf("Invalid method\n");
+
+	//vTaskDelete(NULL);

```

Comment:
Same as above, commented line of code that is not actually a descriptive statement.